run

```diff
@@ -3,5 +3,5 @@ set -ex
 
 docker build --tag webhook .
 docker run --rm \
-    -p 5000:80 \
+    -p 4567:80 \
```

Comment:
This port should be used in the dockerfile as `EXPOSE 4567` (looks like 4567 is not a bad choice: https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers) - I don't see where the port is set in the app though

Reply:
I was just using that port to expose to the world with `ngrok` ;)

Reply:
(The Flask image automatically runs the Flask app on port 80 inside the container)

Reply:
oh of course